src/WelcomeModal.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect} from 'react';
+import Modal from 'react-bootstrap/Modal';
+function WelcomeModal() {
+    const [show, setShow] = useState(false);
+
+  const handleClose = () => {
+    setShow(false);
+    localStorage.setItem('confessit.welcome', true)
+  }
+
+  useEffect(() => {
+    const hasSeenWelcome = localStorage.getItem('confessit.welcome')
+    if (!hasSeenWelcome){
+        setShow(true);
+    }
+  },[])
+
+  return (
+    <>
+      <Modal show={show} onHide={handleClose} size="xl">
+        <Modal.Header closeButton>
+          <Modal.Title>Welcome!</Modal.Title>
+        </Modal.Header>
+        <Modal.Body><span className="text-primary">ConfessIt</span> is a tool to help Roman Catholics walk through an examination of conscience prior to going to confession. 
```

Comment:
Can we use a `Trans` component here to make this a translated string? (English only is fine for now, but putting it in a component will let us add translations for it later. See `About.js` for an example:

https://github.com/kas-catholic/confessit-web/blob/0f6f205b9b86dd1716d68769fc1e5e3fbb726ae3/src/About.js#L16-L18

Reply:
added!